### ISSUE TITLE
strange unstable wav2vec2 pretraining

### ISSUE THREAD

**Issue by Mohammadtvk** [facebookresearch/fairseq#4948]
## ❓ Questions and Help

I am trying to pretrain wav2vec2 on persian language using common voice dataset. I did not modify anything but the dataset path in configs. here is plots of training metrics

![image](https://user-images.githubusercontent.com/23360472/213112672-b0efef76-ba88-4ed7-b581-a57bd97a46f8.png)
![image](https://user-images.githubusercontent.com/23360472/213112830-608474be-1516-4437-ba67-4a023ceb3069.png)
![image](https://user-images.githubusercontent.com/23360472/213112899-15304ce8-0c79-48ec-87ad-4057f79a75f9.png)
![image](https://user-images.githubusercontent.com/23360472/213113003-89eb306b-a93b-4266-943f-48dd834de12e.png)

as can be seen in above figures, after few epochs grad norm and accuracy drops, code perpelexity spike, and loss increases and then stays almost flat. why is that ?

#### What's your environment?

 - fairseq Version (e.g., 1.0 or main): 0.12.2
 - PyTorch Version (e.g., 1.0): 1.13.1
 - OS (e.g., Linux): ubuntu 20.04
 - How you installed fairseq (`pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.8.10
 - CUDA/cuDNN version: 12.0
 - GPU models and configuration: Tesle T4 x2
 - Any other relevant information:

**Comment by tarudesu** [comment]
@Mohammadtvk  Hi there, I have the same problem with you. Could I know that how can you fix that one, please?

**Comment by StudentofSong** [comment]
Hi there, I have the same problem with you. Have you solved your problem.Could I know that how can you fix that one, please?
